_posts/2016-05-20-drupalcon-nola-recap.md

```diff
@@ -0,0 +1,118 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<span class="caption">...Only at Drupalcon...</span>
+
+### Tuesday - Presentation / Women in Drupal
+
+Excited (and a bit nervous) to give [my first presentation at Drupalcon](https://events.drupal.org/neworleans2016/sessions/total-value-ownership-drupal-8-and-beyond) (e pluribus unum), I spent much of the day reviewing and updating slides that were impacted by [the Driesnote](https://www.youtube.com/watch?v=Ksp5JVFryEg) which had a lot of overlap with my presentation.
+
+Since Drupalcon is primarily a developer's conference, and I was speaking in the "Business" track, I decided to drum up some interest for the session with a little social media <s>mudslinging</s> banter.
+
+A couple favorites are:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Is <a href="https://twitter.com/acquia">@acquia</a>&#39;s <a href="https://twitter.com/HornCologne">@horncologne</a> secretly involved with <a href="https://twitter.com/Jamroom">@Jamroom</a> CMS?<br><br>Sifting through the rumors today <a href="https://twitter.com/5">@5</a>:00 <a href="https://twitter.com/DrupalConNA">@DrupalConNA</a> <a href="https://t.co/T4sQ5Pmbih">https://t.co/T4sQ5Pmbih</a></p>&mdash; Savas Labs (@Savas_Labs) <a href="https://twitter.com/Savas_Labs/status/730096335748206592">May 10, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+I must admit, since we have a [Jam](/team/kosta-harlan) of our own (inquire within), Acquia's Jam is a topic of frequent discussion at the Savas Labs water cooler. I <s>was</s> still am hopeful for a reaction.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">TBD today <a href="https://twitter.com/5">@5</a>:00 <a href="https://twitter.com/DrupalConNA">@DrupalConNA</a> <a href="https://t.co/VODC0fZ65P">https://t.co/VODC0fZ65P</a><br><br>What&#39;s the best/worst named CMS?<br>Farcry<br>Geeklog<br>Thingamablog<a href="https://twitter.com/TiddlyWiki">@TiddlyWiki</a> (my fave)</p>&mdash; Savas Labs (@Savas_Labs) <a href="https://twitter.com/Savas_Labs/status/730087801790828545">May 10, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Just a select few of the **incredible** names _someone_ decided to name their CMS in the past. Most of which have little to no presence at this point. You live and learn.
```

Comment:
Is this supposed to be a caption under the image? If not, it seems out of context.